Subsystem debug

MatthewHinspeter opened the pull request.

MatthewHinspeter committed:
Added NykoController support. Converted existing OperatorController to NykoController.

MatthewHinspeter committed:
Implemented manual override roller test functionality.

MatthewHinspeter committed:
Added storage sensor display.

ajwharton reviewed (commented):
Comments would be great for all of this.

jhinspeter committed:
Merge remote-tracking branch 'remotes/origin/master' into Subsystem_Debug

# Resolved Conflicts:
#	src/main/java/frc/robot/OI/OperatorInterface.java

Commit:
Google Java Format

MatthewHinspeter committed:
Added config for motors and sensors.
Updated harvest mode to use the operator controller.
Removed manual override button, moved test functions to testPeriodic.
Changed Intake sensor from analog input to digital input.

MatthewHinspeter committed:
Merge branch 'Subsystem_Debug' of https://github.com/team138Entropy/Entropy2020 into Subsystem_Debug

# Resolved Conflicts:
#	src/main/java/frc/robot/OI/OperatorInterface.java
#	src/main/java/frc/robot/Robot.java

Commit:
Google Java Format

MatthewHinspeter committed:
Merge remote-tracking branch 'remotes/origin/master' into Subsystem_Debug

# Resolved Conflicts:
#	src/main/java/frc/robot/subsystems/Shooter.java

MatthewHinspeter committed:
Merge branch 'Subsystem_Debug' of https://github.com/team138Entropy/Entropy2020 into Subsystem_Debug

Commit:
Google Java Format

MatthewHinspeter committed:
Removed linter

MatthewHinspeter committed:
Merge branch 'Subsystem_Debug' of https://github.com/team138Entropy/Entropy2020 into Subsystem_Debug

ajwharton commented on src/main/java/frc/robot/OI/OperatorInterface.java:
```diff
@@ -69,48 +70,53 @@ public boolean CheckLowGear(boolean previous) {
-    return OperatorController.getJoystick(XboxController.Side.RIGHT, XboxController.Axis.X);
-  }
-
-  public int getTurretManual() {
```
Did we delete the method getTurretManual?

jhinspeter replied:
Replaced by `getTurretAdjustLeft()` and   `getTurretAdjustRight()`
